### ISSUE TITLE
Weird line breaks on https://chairs.ietf.org/documents/qa-style-writeup-template

### ISSUE THREAD

**Issue by larseggert** [ietf-chairs/chairs.ietf.org#6]
There are some really weird linebreaks on https://chairs.ietf.org/documents/qa-style-writeup-template, especially inside hyperlinks?

**Comment by ghwood** [comment]
Yes there were some hard line breaks in the markdown which introduced funkiness. Not sure how they got there but they are now gone.
